surfactant/cmd/generate.py

```diff
@@ -431,10 +431,16 @@ def sbom(
                                 except Exception as e:
                                     raise RuntimeError(f"Unable to process: {filepath}") from e
 
+                        if not entry.includeFileExts:
+                            entry.includeFileExts = []
+                        if not entry.excludeFileExts:
+                            entry.excludeFileExts = []
```

Comment:
I was debating if the check should happen in `surfactant/filetypeid/id_extension.py` -- but I think this is the only place that makes sense, since it needs to be able to exclude files by extension even if they were identified by something like magic bytes.

We could perhaps expand `Exts` to `Extensions`, though `Ext` is a common enough abbreviation.

Reply:
The only other thought I have about this is should we add a core config setting that lets a user set a default list of include/exclude file exts?

Reply:
Yeah, since a file could be identified using magic bytes I'm not sure where else it could go.
I think adding a core setting in addition to the list could work, but then you run into if it overrides or adds to the lists, which may cause confusion.